build/source/netcdf/def_output.f90

```diff
@@ -132,7 +132,8 @@ subroutine def_output(nHRU,nSoil,infile,err,message)
    ! error handling
    if(err/=0)then;err=20;message=trim(message)//trim(cmessage)//'[structure =  '//trim(structInfo(iStruct)%structName);return;end if
   end do ! iStruct 
-
+  ! write HRU dimension for each output file
+  call write_hru_dim(ncid(iFreq), err, cmessage); if(err/=0) then; message=trim(message)//trim(cmessage); return; end if
```

Comment:
was trying to keep it consistent with other function names in this file since it is an internal subroutine being used locally.